ports/activemq-cpp/vcpkg.json

```
@@ -1,9 +1,9 @@
 {
   "name": "activemq-cpp",
-  "version-string": "3.9.5",
+  "version-semver": "3.9.5",
   "port-version": 4,
```

```suggestion
  "version": "3.9.5",
  "port-version": 5,
```

1. I don't see documentation from these folks claiming that they follow semver?
2. port-version should be updated.

---

Apache doesn't do full semver but it does follow a fully compliant subset: https://apr.apache.org/versioning.html